Normalization: Use default access rule if normal rule errors

Hwesta opened the pull request:
If the access normalization rule fails, we should fall back to the default access rule to ensure there is always an access copy. Not needed for preservation, since the original is always in the AIP.

refs redmine 10779

Hwesta committed:
Normalization: Use default access rule if normal rule errors

If the access normalization rule fails, we should fall back to the default
access rule to ensure there is always an access copy. Not needed for
preservation, since the original is always in the AIP.

refs #10779

sevein reviewed (approved):
I locally updated `archivematicaClient.py` to make sure that the stdout of client scripts was being redirected to the master process. That helped to verify that your change was working properly, this is what I saw:

```
Command exit code: -1
access normalization failed, falling back to default access rule
Fallback Format Policy Rule: Normalize Unknown: Default Format: Unknown file format for Default Access via Copying file to access directory
Fallback Format Policy Command Copying file to access directory
Command to execute: cp -R "/var/archivematica/sharedDirectory/currentlyProcessing/Prueba003-55ad876b-8224-41fe-9f95-39e1af4db628/objects/lzone.jpg" "/var/archivematica/sharedDirectory/currentlyProcessing/Prueba003-55ad876b-8224-41fe-9f95-39e1af4db628/DIP/objects/e1ecf3e1-494d-4380-9962-ef747abbe3b7-lzone.jpg"
```

jrwdunham committed:
Use default access rule only if access derivative not exists

jrwdunham committed:
Normalization: Use default thumbnail rule if normal rule errors

The error-then-default behaviour for access derivatives needed to be
ported to thumbnails also. If not, then a transfer containing a single
file that errors on thumbnail creation (with no thumbnail created) will
cause a failed SIP at Prepare DIP: Copy thumbnails to DIP directory
because the thumbnails/ dir will be empty and will have been removed,
causing `cp` to error. Refs #10779.

mcantelon commented:
Looks good to me. Nice comment!

jhsimpson commented:
rebased onto most recent qa/1.x

jhsimpson committed:
fix flake8 error

E201 whitespace after '(' reported by travis.
